data/analytics/events.yml

```diff
@@ -384,7 +384,7 @@
         - name: event_name
           value: navigation
         - name: external
-          value: true
+          value: string (true)
```

Comment:
Although this isn't a new entry for some reason the `external` attribute isn't showing on the current deployment, we might need to check.

Reply:
Yep, it doesn't display because just putting `true` makes it a `boolean` instead of a `string` value.